src/app/screens/BookDetail/components/Suggestions/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import dataBooks from '../../../../../data/books.json';
+
+import SuggestionsLayout from './layout';
+
+class Suggestions extends React.Component {
+  state = { books: [] };
+
+  componentWillMount = () => {
+    this.setState({ books: this.getRandomSuggestions() });
+  };
```

Comment:
`componentWillMount` will be deprecated.

Read this https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html and try to replace those functions with the new ones!